src/components/TodoContainer.js

```diff
@@ -75,32 +74,59 @@ export default function TodoContainer() {
         id: jsonResponse.id,
       };
 
-      console.log(airtableTodo);
       setTodoList([...todoList, airtableTodo]);
     } catch (error) {
       console.log(error.message);
     }
   };
-  const removeTodo = (id) => {
-    const removeItem = todoList.filter((todo) => todo.id !== id);
 
-    setTodoList(removeItem);
+  const removeTodo = async (id) => {
+    try {
+      const airtableData = {
+        fields: {
+          id: id,
+        },
+      };
+      const options = {
+        method: "DELETE",
+        headers: {
+          "Content-Type": "application/json",
+          Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`,
+        },
+        body: JSON.stringify(airtableData),
+      };
+
+      const url = `
+https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}/${id}`;
+
+      const response = await fetch(url, options);
```

Comment:
You're currently ignoring the response from Airtable, which will tell you if the item was successfully deleted from the backend. 

You can handle this in a similar way you did for adding a new todo: 

```
      if (!response.ok) {
        const message = `Error: ${response.status}`;
        throw new Error(message);
      }
```